examples/private_set_intersection/client/private_set_intersection.cc

```diff
@@ -78,14 +81,36 @@ int main(int argc, char** argv) {
 
   // Submit sets from different clients.
   std::vector<std::string> set_0{"a", "b", "c"};
-  SubmitSet(stub_0.get(), set_0);
+  auto submit_status_0 = SubmitSet(stub_0.get(), set_0);
+  if (!submit_status_0.ok()) {
+    LOG(FATAL) << "Could not submit set: " << submit_status_0.error_code() << ": "
+               << submit_status_0.error_message();
+  }
 
   std::vector<std::string> set_1{"b", "c", "d"};
-  SubmitSet(stub_1.get(), set_1);
+  auto submit_status_1 = SubmitSet(stub_1.get(), set_1);
+  if (!submit_status_1.ok()) {
+    LOG(FATAL) << "Could not submit set: " << submit_status_1.error_code() << ": "
+               << submit_status_1.error_message();
+  }
 
-  std::set<std::string> expected_set{"b", "c"};
+  // Use an invalid public key.
+  std::string invalid_public_key_base64 = "vpxqTZOUq1FjcaB9uJYCuv4kAg+AsgMwubA6WE+2pmk=";
+  std::string invalid_public_key;
+  if (!absl::Base64Unescape(invalid_public_key_base64, &invalid_public_key)) {
+    LOG(FATAL) << "Could not decode public key: " << invalid_public_key_base64;
+  }
+  oak::label::Label invalid_label = oak::WebAssemblyModuleSignatureLabel(invalid_public_key);
+  auto invalid_stub = PrivateSetIntersection::NewStub(oak::ApplicationClient::CreateChannel(
+      address, oak::ApplicationClient::GetTlsChannelCredentials(ca_cert), invalid_label));
+  std::vector<std::string> set_2{"c", "d", "e"};
+  auto submit_status_2 = SubmitSet(invalid_stub.get(), set_2);
+  if (submit_status_2.ok()) {
```

Comment:
Could you make this match a bit more specific? Currently it would probably succeed if there is no connectivity, for instance.

Reply:
Done